charts/gpu-metrics-exporter/values.yaml

```diff
@@ -31,14 +22,21 @@ gpuMetricsExporter:
   securityContext:
     readOnlyRootFilesystem: true
     runAsNonRoot: true
-  volumeMounts: []
+  port: 6061
 
 dcgmExporter:
   enabled: true
-  arguments: ["-f", "/etc/dcgm-exporter/dcp-metrics-included.csv"]
   image:
     repository: nvcr.io/nvidia/k8s/dcgm-exporter
     pullPolicy: IfNotPresent
-    tag: 3.3.6-3.4.2-ubuntu22.04
+    tag: 3.3.7-3.5.0-ubuntu22.04
   useExternalHostEngine: false
-  additionalEnv: []
+
+gke:
```

Comment:
https://github.com/castai/gpu-metrics-exporter/blob/8ee5cacbdcc247bbf40f2167995b04c7875a1f6c/charts/gpu-metrics-exporter/values-gke.yaml won't this work?

Reply:
i want to move away from multiple files to simplify the install script we have in kubecast

with the multiple files we need to pull and untar the chart before install